docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/03-Digital-Ocean-(manual).md

```diff
@@ -5,36 +5,42 @@ description: Learn how to deploy your Prisma database service to Digital Ocean m
 
 # Digital Ocean (manual)
 
-<InfoBox type=warning>
+This section describes how to set up a fully functioning Prisma server on Digital Ocean in less than 20 minutes.
+
+In the following procedure you will connect to your droplet with ssh, and manually install all required dependencies. Most users should use the procedure described in [Digital Ocean (Docker Machine)](!alias-texoo9aemu) instead.
+
+[Digital Ocean](https://www.digitalocean.com/) is an easy-to-use provider of virtual servers. They offer configurable compute units of various sizes, called [Droplets](https://www.digitalocean.com/products/droplets/).
 
-**This tutorial only applies to Prisma versions lower than 1.7 and will be updated soon.**
+<InfoBox>
+
+The setup described in this section does not include features you would normally expect from production-ready servers, such as _automated backups_ and _active failover_. We will add more guides in the future describing a more complete setup suitable for production use.
 
 </InfoBox>
 
-This section describes how to set up a fully functioning Prisma server on Digital Ocean in less than 20 minutes.
+## 1. Register at Digital Ocean
 
-In the following procedure you will connect to your droplet with ssh, and manually install all required dependencies. Most users should use the procedure described in [Digital Ocean (Docker Machine)](!alias-texoo9aemu) instead.
+<Instruction>
+
+If you haven't already, you need to start by creating an account at for Digital Ocean. You can do so [here](https://cloud.digitalocean.com/registrations/new).
 
-Digital Ocean is an easy to use provider of virtual servers. They offer different compute sizes called droplets. The 10$ droplet is a good starting point for a Prisma server.
+</Instruction>
 
-> The setup described in this section does not include features you would normally expect from production ready servers, such as backup and active failover. We will add more guides in the future describing a more complete setup suitable for production use.
 
-## Creating a droplet
+## 2. Create your Digital Ocean Droplet
 
-If you are new to Digital Ocean you should start by creating an account at https://www.digitalocean.com/
+You can now go ahead and create your Droplet. First pick the latest Ubuntu Distribution, as of this writing it is `18.04.1 x64`. Next pick a region that is close to your end user. If this Prisma will be used for development, you should pick the region closest to you. Set a name for your droplet, `prisma` for example.
```

Comment:
This doesn't make clear where/how I need to go to create the Droplet. Is there a URL or some other hint that can be provided here? Might also be helpful to include a screenshot of the website (side-note re images: we're usually hosting them on imgur.com and then referencing them there).

Reply:
"If this Prisma will be used for development, you should pick the region closest to you."

1. why only in development? 
2. here we should talk about a **Prisma server** instead of **a Prisma**